ui/helpers/constants/settings.js

```diff
@@ -266,67 +266,67 @@ export const SETTINGS_CONSTANTS = [
     sectionMessage: (t) => t('metamaskVersion'),
     descriptionMessage: (t) => t('builtAroundTheWorld'),
     route: `${ABOUT_US_ROUTE}#version`,
-    icon: 'fa fa-info-circle',
+    iconName: ICON_NAMES.INFO,
```

Comment:
Has the settings page been update to allow for this? I assume so

Reply:
Yup, we already have a use-case for settings icon in this file that's using iconName ` iconName: ICON_NAMES.SETTING,`